File: C++/C++Basic/09.While-Loop-Lab/05.account-balance.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string command;
    cin >> command;

    double totalSum = 0;

    cout.setf(ios::fixed);
    cout.precision(2);
    
    while (command != "NoMoreMoney")
    {
        double money = stod(command);

        if (money < 0)
        {
            cout << "Invalid operation!" << endl;
            break;
        }
        else
        {
            cout << "Increase: " << money << endl;
        }

        totalSum += money;

        cin >> command;
    }

    cout << "Total: " << totalSum << endl;
    return 0;
}

File: Python/Basic/09.While-Loop-Lab/05.account_balance.py
command = input()

totalSum = 0

while command != 'NoMoreMoney':
    money = float(command)
    if money < 0:
        print('Invalid operation!')
        break
   
    print(f'Increase: {money:.2f}')
   
    totalSum += money

    command = input()

print(f'Total: {totalSum:.2f}')